packages/fxa-shared/lib/logger.ts

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Logging module that performs proper checks and string
+ * interpolation before logging message.
+ *
+ */
+
+// tslint:disable-next-line: no-var-requires
+// require('jsdom-global')();
```

Comment:
What's this?

Reply:
Ah I'll remove. Leftover from testing something